web/assets/css/transmission-app.scss

```diff
@@ -322,7 +322,8 @@ $speed-icon-size: 20px;
 #speed-dn-label,
 #speed-up-label {
   text-align: right;
-  width: 70px;
+  width: 100px;
```

Comment:
Maybe the `width` property should be removed. The speed text and icon will move around as the speed changes, but I think this looks better than the big blank space.

See what you think.

#### Before

![image](https://github.com/transmission/transmission/assets/46261767/5def791b-3881-4aca-8750-c231f811758e)

#### After

![image](https://github.com/transmission/transmission/assets/46261767/6c8049e0-e9ea-45e4-a99a-89c977a83cf2)

Reply:
Nope, I think I'd rather have a static width. If the text always moves around it is less legible and it takes for your brain more time to process the info.

Reply:
Then maybe make it narrower?

The biggest unit possible here is MB/s. Having over 1000 MB/s transfer speed is not impossible, but quite a far fetch, so I think we can assume `xxxx.xx MB/s` is the widest string possible.

Reply:
The entire idea of this PR was to not wrap and use 2 rows. I've tested 100px and I never ran into any issue ever again. 70 vs 100 may seem a huge increase percentage wise, but what are 100px these days?
Are you running the web interface on a 300px device?
I haven't used a display less than 1000px for more than 10 years. Even my Pixel 5 has 1080 x 2340 pixels.

Btw, `xxxx.xx MB/s` is actually wider than 100px.